src/components/panels/LocalAppsPanel/LocalAppsPanel.js

```diff
@@ -19,86 +19,103 @@ import {
   filterModelStatusDataByApp,
 } from "app/utils";
 
+import ConfigPanel from "../ConfigPanel/ConfigPanel";
+
 import "../_panels.scss";
+import "./_local-apps-panel.scss";
+
+// Generate panel header for given entity
+const generateAppPanelHeader = (app, title, showConfig) => {
+  return (
+    <div className="panel-header">
+      {app && (
+        <div className="row">
+          <div className="col-4">
+            <div>{title}</div>
```

Comment:
This should probably be a heading element.

Reply:
The `title` prop contains a number of components as it reuses the heading renderer from the other panel headings.